doc/mgr/orchestrator.rst

```diff
@@ -486,6 +487,8 @@ where the properties of a service specification are:
     a string. Use '\n' for line breaks in that case. Otherwise define
```

Comment:
(Not  incurred by this PR) The `\n` doesn't seem to render well in GitHub and Sphinx html output. Could you fix it together? Thanks.

![Screenshot_20201019_165739](https://user-images.githubusercontent.com/1691518/96423756-5cd11780-122c-11eb-8e3b-800d449a8b3e.png)